include/clink/linalg/sparse_vector.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CLINK_LINALG_SPARSE_VECTOR_H_
+#define CLINK_LINALG_SPARSE_VECTOR_H_
+
+#include "llvm/ADT/SmallVector.h"
+
+namespace clink {
+
+// A sparse vector of double values.
+class SparseVector {
+public:
+  SparseVector(const int n);
```

Comment:
Could we have empty lines between these methods to follow TFRT's code style (which I believe is also Google's code style)?

And since this is a infra class that could be used by many algorithms in the future, could we have add doc for these methods?

Reply:
OK. I'll make the changes.